### DISCUSSION TITLE
Stuck on how to re-authorize the application

### DISCUSSION THREAD

**Discussion by steveklett** [abraunegg/onedrive#3021] [Q&A] [answered]
I'm on section 5 of [How to configure OneDrive SharePoint Shared Library sync](https://github.com/abraunegg/onedrive/blob/master/docs/sharepoint-libraries.md) and there is an info bubble that states:

_"As this is a new configuration, the application will be required to be re-authorised the first time this command is run with the new configuration."_

When I execute the -`-dry-run` I do indeed see messages stating an authorization is required, but it's unclear (to me) what I'm supposed to do.

```
steve@steve-Latitude-E7470:~$ onedrive --confdir="~/Documents/xxx/.config/xxx" --dry-run
Reading configuration file: /home/steve/Documents/xxx/OneDrive/.config/xxx/config
Configuration file successfully loaded
D-Bus message bus daemon is available; GUI notifications are now enabled
DRY-RUN Configured. Output below shows what 'would' have occurred.
Using IPv4 and IPv6 (if configured) for all network operations
Attempting to contact Microsoft OneDrive Login Service
Successfully reached Microsoft OneDrive Login Service
Configuring Global Azure AD Endpoints

The application requires authorisation, which involves saving authentication data on your system. Note that authorisation cannot be completed with the '--dry-run' option.

To exclusively authorise the application without performing any additional actions, use this command: onedrive
```
As a long-shot, I tried the simple command `onedrive` without any arguments, but that didn't work.

I've looked through the documentation and I can't find anything about what I need to do to re-authorize the application.

**Comment by abraunegg** [comment, marked as answer]
Please use your original command without `--dry-run` to authenticate the application

**Reply by steveklett** [reply]
That worked to trigger the auth, thank you.
Note: I don't seem to have a way to mark your comment as the answer, the option isn't there. /shrug
